gap/PolygonalComplexes/modification.gd

```diff
@@ -167,7 +167,21 @@
 #! replaced by the appropriate number of new labels. The new labels can be
 #! defined by the optional argument <A>newEdgeLabels</A>.
 #!
-#! TODO examples
+#! For example consider the following triangular complex:
+#! <Alt Only="TikZ">
+#!	\input{Image_Butterfly.tex}
```

Comment:
The two pictures are differently oriented (rotated and reflected). They are also differently sized. This picture should be rotated and appropriately labelled.